projects/openapi-cli/src/commands/reporters/feedback.ts

```diff
@@ -1,14 +1,13 @@
 import { Command } from 'commander';
 import { Writable } from 'stream';
 import { trackEvent, flushEvents } from '../../segment';
-
+import chalk from 'chalk';
 export async function createCommandFeedback(
```

Comment:
This doesn't need to be async - which I think is causing all the other commands to be "async"

Reply:
good catch. We changed Chalk to the version that's compatible with our code so we don't need to dynamically import it anymore. I've updated all the usages / downstreams